app/javascript/scimo/index.js

```diff
@@ -10,10 +10,13 @@ import "./styles/StatesOfMatter.css";
 import "./styles/CellsTissuesOrgans.css";
 import "./styles/lesson_template.css";
 import "./styles/SolarSystem.css";
+import "./styles/AlkaliMetals.scss";
 
 
 import "./scripts/DNA.js";
 import "./scripts/Sound.js";
 import "./scripts/Eye.js";
 import "./scripts/Atoms.js";
 import "./scripts/smoothscroll.js";
+import "./scripts/three.js";
```

Comment:
I think this can also be yarn installed like the d3 library

`yarn add three`
`import * as THREE from 'js/three.module.js';`

Reply:
Oh, we did yarn install it.  I guess we didn't need this script in addition.  I'll remove it.